exercises/concept/tim-from-marketing/src/test/java/BadgeTests.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BadgeTests {
+
+    @Test
+    @Tag("task:1")
+    @DisplayName("The print method of the TimFromMarketing class returns the correct label for the employee")
```

Comment:
This name is a bit verbose and actually confusing as the class is not called `TimForMarketing`. I suggest simplifying the display names in this file. For example:

1. Printing a badge for an employee
2. Printing a badge for a new employee
3. Printing a badge for the owner
4. Printing a badge for the owner who is a new employee